doc/README.md

```diff
@@ -33,11 +33,21 @@ Give us a star if you want to help give the engine exposure and grow the communi
 
 ## Installation
 
-Put the pub package as your dependency by putting the following in your `pubspec.yaml`:
-
-```yaml
-dependencies:
-  flame: --VERSION--
+Add the `flame` package as a dependency in your `pubspec.yaml` by running the following command:
+
+```console
+$ flutter pub add flame
+Resolving dependencies...
```

Comment:
I think we can skip the output of the command, it might confuse some beginners since it'll say different versions for them later

Reply:
The linter spat the dummy because I had the `$` before the command and no output. I can strip it back to just the command without upsetting the linter?